bioformats/tests/test_formatreader.py

```diff
@@ -119,25 +120,58 @@ def test_03_02_load_using_bioformats(self):
              [5, 2, 3, 3, 2, 2, 2, 3, 2, 2]], dtype=np.uint8)
         self.assertTrue(np.all(expected_0_10_0_10 == data[:10,:10]))
         self.assertTrue(np.all(expected_n10_n10 == data[-10:,-10:]))
+
+    def test_03_03_read_subimage_tif(self):
+        path = os.path.join(os.path.dirname(__file__), 'Channel1-01-A-01.tif')
+        with bioformats.ImageReader(path) as f:
+            data_0_10_0_10 = f.read(XYWH=(0, 0, 10, 10), rescale=False)
+
+        #
+        # Data as read by cellprofiler.modules.loadimages.load_using_PIL
+        #
+        expected_0_10_0_10 = np.array(
+            [[ 0,  7,  7,  6,  5,  8,  4,  2,  1,  2],
+             [ 0,  8,  8,  7,  6, 10,  4,  2,  2,  2],
+             [ 0,  9,  9,  7,  8,  8,  2,  1,  3,  2],
+             [ 0, 10,  9,  8, 10,  6,  2,  2,  3,  2],
+             [ 0, 10, 10, 10,  9,  4,  2,  2,  2,  2],
+             [ 0,  9,  9, 10,  8,  3,  2,  4,  2,  2],
+             [ 0,  9,  9, 10,  8,  2,  2,  4,  3,  2],
+             [ 0,  9,  8,  9,  7,  4,  2,  2,  2,  2],
+             [ 0, 10, 11,  9,  9,  4,  2,  2,  2,  2],
+             [ 0, 12, 13, 12,  9,  4,  2,  2,  2,  2]], dtype=np.uint8)
+        expected_n10_n10 = np.array(
+            [[2, 1, 1, 1, 2, 2, 1, 2, 1, 2],
+             [1, 2, 2, 2, 2, 1, 1, 1, 2, 1],
+             [1, 1, 1, 2, 1, 2, 2, 2, 2, 1],
+             [2, 2, 2, 2, 3, 2, 2, 2, 2, 1],
+             [1, 2, 2, 1, 1, 1, 1, 1, 2, 2],
+             [2, 1, 2, 2, 2, 1, 1, 2, 2, 2],
+             [2, 2, 3, 2, 2, 1, 2, 2, 2, 1],
+             [3, 3, 1, 2, 2, 2, 2, 3, 2, 2],
+             [3, 2, 2, 2, 2, 2, 2, 2, 3, 3],
+             [5, 2, 3, 3, 2, 2, 2, 3, 2, 2]], dtype=np.uint8)
+        self.assertTrue(np.all(expected_0_10_0_10 == data_0_10_0_10))
+        # self.assertTrue(np.all(expected_n10_n10 == data[-10:,-10:]))
         
-    def test_03_03_load_using_bioformats_url(self):
-        data = F.load_using_bioformats_url(
-            "http://www.cellprofiler.org/linked_files/broad-logo.gif",
-            rescale=False)
-        self.assertSequenceEqual(data.shape, (38, 150, 3))
-        expected_0_10_0_10 = np.array([
-            [181, 176, 185, 185, 175, 175, 176, 195, 187, 185],
-            [ 25,   7,   7,   7,   2,   2,  13,  13,   0,   1],
-            [ 21,   1,   1,   0,   0,   1,   0,   1,   0,   0],
-            [ 64,  13,   1,   1,  12,  12,   2,   1,   1,   1],
-            [ 22,  56,  26,  13,   1,   1,   6,   0,   0,   0],
-            [ 12,  13,  82,  57,   9,  12,   2,   6,   6,   6],
-            [ 12,  13,  20,  89,  89,  21,  11,  12,   1,   0],
-            [  6,   1,   7,  21,  89, 102,  26,   0,  10,   1],
-            [ 26,   0,   0,   1,  20,  84, 151,  58,  12,   1],
-            [ 23,   6,   1,   1,   0,   1,  55, 166, 100,  12]], 
-                                      dtype=np.uint8)
-        self.assertTrue(np.all(expected_0_10_0_10 == data[:10,:10, 0]))
+    # def test_03_03_load_using_bioformats_url(self):
```

Comment:
I'm going to accept the PR and then fix this on my own. Deck chairs have moved on the website - the easiest thing is to point the URL to https://github.com/CellProfiler/python-bioformats/raw/1.0.5/bioformats/tests/Channel1-01-A-01.tif and that is stable going forward. The test should check to make sure it can read the header at the URL w/o 404 in case Github disappears in the year 3000 AD and someone is still using this code.